src/commands/createManagedEnvironment/LogAnalyticsCreateStep.ts

```diff
@@ -19,7 +19,7 @@ export class LogAnalyticsCreateStep extends ExecuteActivityOutputStepBase<IManag
     protected async executeCore(context: IManagedEnvironmentContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
         const opClient = await createOperationalInsightsManagementClient(context);
         const resourceGroup = nonNullProp(context, 'resourceGroup');
-        const workspaceName = nonNullProp(context, 'newManagedEnvironmentName');
+        const workspaceName = context.newLogAnalyticsWorkspaceName || nonNullProp(context, 'newManagedEnvironmentName');
```

Comment:
Should we fallback to the `newManagedEnvironmentName`? If there isn't a `newLogAnalyticsWorkspaceName`, couldn't that mean that a log analytics workspace wasn't actually created?

Reply:
I'm not sure I understand - are you asking in the sense that you don't think we should be falling back to the `newManagedEnvironmentName`?  Because I am falling back to that value, but as a `nonNullProp`

Reply:
Sorry, yeah I meant more like should we be doing that because I don't think it makes sense for the reasons stated above.

Reply:
Yeah maybe not, but that's how it was already designed.  I didn't want to rework the entire managed environment/log analytics design.  Maybe that's a future todo though.